blocks/explore-cards/explore-cards.css

```diff
@@ -1 +1,194 @@
-/* TBD */
\ No newline at end of file
+.explore-cards-wrapper {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.explore-cards {
+    width: 100%;
+    padding: var(--spacing-48) 0 var(--spacing-64) 0;
+}
+
+.heading {
+    margin-bottom: var(--spacing-20);
+}
+
+.heading p {
+    font-size: var(--heading-font-size-m);
+    line-height: 30.8px;
+}
+
+.explore-card-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: var(--spacing-20);
+    width: calc(100vw - 68px);
+    overflow-x: scroll;
+}
+
+.explore-card {
+    position: relative;
+    border-radius: 10px;
+    height: 400px;
+    min-width: 300px;
+}
+
+.card-icon.icon-video {
```

Comment:
card icons has all common properties except the icon. add common properties under card-icon
```
.card-icon {
    width: 72px;
    height: 72px;
    background-color: var(--background-color);
    border-radius: 50%;
    display: flex;
    justify-content: center;
    align-items: center;
    margin: 20px 0;
    background-repeat: no-repeat;
    background-position: center;
}

.card-icon.icon-video {
    background-image: url("/icons/playbutton.svg");
}

.card-icon.icon-audio {
    background-image: url("/icons/podcast.svg");
}
```